packages/annotations/package.json

```diff
@@ -29,7 +29,7 @@
   ],
   "dependencies": {
     "@nivo/colors": "0.80.0",
-    "@react-spring/web": "9.4.5",
+    "@react-spring/web": "9.4.5 || ^9.7.2",
```

Comment:
The versions >9.4.5 and <9.7.2 are not valid from a TS point of view, thus I dropped them from the range of acceptable versions